crates/uv/src/commands/python/install.rs

```diff
@@ -80,6 +81,24 @@ impl InstallRequest {
     fn matches_installation(&self, installation: &ManagedPythonInstallation) -> bool {
         self.download_request.satisfied_by_key(installation.key())
     }
+
+    fn managed_download_matches_installation(
+        &self,
+        installation: &ManagedPythonInstallation,
+    ) -> bool {
+        self.download.key() == installation.key()
+    }
+
+    fn is_version_request(&self) -> bool {
+        matches!(self.request, PythonRequest::Version(..))
+    }
+
+    fn is_minor_request(&self) -> bool {
+        matches!(
+            self.request,
+            PythonRequest::Version(VersionRequest::MajorMinor(..))
+        )
+    }
```

Comment:
I'm unsure of the value of these. Is this just for readability of `request.is_version_request() && !request.is_minor_request())`? I'm generally hesitant to add indirection for something only used once.